### DISCUSSION TITLE
Pause on focus loss

### DISCUSSION THREAD

**Discussion by beam41** [OpenRCT2/OpenRCT2#18112] [Feature requests]
Many games pause when you lose focus like when you are switching windows. This would be a great feature for me who 'alt-tab' a lot but doesn't want to leave the game unattended.

**Comment by karst** [comment]
I personally have shortcut bound my pause to spacebar. This is how i am used to it from games like cities: skylines. These games also continue running when alt tabbed, so you can afk for a bit if you want to and you can easily pause to check something real quick.

**Comment by ocalhoun6** [comment]
Should be optional, though. For people who want the ability to let a park run in the background while doing other things.

**Reply by beam41** [reply]
Yes, like the steam overlay pause option.
